cli/types/tests/cypress-tests.ts

```diff
@@ -338,3 +338,15 @@ namespace CypressLocationTests {
   cy.location('path') // $ExpectError
   cy.location('pathname') // $ExpectType Chainable<string>
 }
+
+namespace CypressBrowserTests {
+  Cypress.isBrowser('chrome')// $ExpectType boolean
+  Cypress.isBrowser('firefox')// $ExpectType boolean
+  Cypress.isBrowser('edge')// $ExpectType boolean
+  Cypress.isBrowser('brave')// $ExpectType boolean
```

Comment:
I would add `Cypress.isBrowser('safari') // $ExpectError` and `Cypress.isBrowser('IE11') // $ExpectError`

Reply:
well, we don't currently error on those. the way the typedefs are made allow `| string`